instrumentation/opentelemetry-instrumentation-celery/src/opentelemetry/instrumentation/celery/__init__.py

```diff
@@ -108,7 +108,23 @@ def _trace_prerun(self, *args, **kwargs):
 
         logger.debug("prerun signal start task_id=%s", task_id)
 
-        span = self._tracer.start_span(task.name, kind=trace.SpanKind.CONSUMER)
+        (parent, publisher) = utils.retrieve_span_context(task)
+
+        links = []
+        if publisher is not None:
+            links.append(trace.Link(publisher))
```

Comment:
I know some tracing backends do not support links properly or at all and suggest using parent<>child relation between publishers and consumers. Can we have a configuration option to let users decide if they want to use links/references or parent<>child relation? I know spec suggests to use links for this but still wondering if it would be possible to use parent<>child as an opt-in behavior.